portal-api/test/06_subscriptionsTest.js

```diff
@@ -1004,25 +1037,24 @@ describe('/applications/<appId>/subscriptions', function () {
             }
         });
 
-        it('should, return the list ordered by application id', function (done) {
+        it('should, return the list ordered by application name', function (done) {
```

Comment:
Why did you change this test case? Sorting by name sounds like an additional test case?

Reply:
In UI we only expose application name and application id